pages/index.tsx

```diff
@@ -16,19 +15,90 @@ const IndexPage: NextPage = () => (
         welcomeMessage="selamat datang"
```

Comment:
```suggestion
        welcomeMessage="Selamat Datang"
```
Proper capitalisation.